client/components/Review.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+// import styles from '/Users/ryanbrennan/Desktop/FEC/reviews-service/client/css/style.css'
+
+class Review extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      username: '',
+      trainame: '',
+      url: ''
+    };
+  }
+
+  componentDidMount() {
+    fetch(
+      `http://localhost:3002/user/${this.props.info.data.attributes.user_id}`
+    )
+      .then(res => {
+        return res.json();
+      })
+      .then(profile => {
+        this.setState({
+          username:
+            profile.data.attributes.first_name +
+            ' ' +
+            profile.data.attributes.last_name,
+          url: profile.data.attributes.photo_url
+        });
+      });
+  }
+
+  render() {
+    return (
+      <div>
+        <title>Review </title>
+        <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css" />
+        <link
+          rel="stylesheet"
+          type="text/css"
+          href="css/font-awesome.min.css"
+        />
+        <link href="css/css.css" rel="stylesheet" />
+        <link rel="stylesheet" type="text/css" href="css/style.css" />
+        <link rel="stylesheet" type="text/css" href="css/media.css" />
+        <div className="review_box_main boxs">
+          <div className="container">
+            <div className="review_box_main_inner boxs">
+              {/*  first box start */}
+              <div className="review_box_main_inner1 boxs">
+                <div className="review_box_main_inner1_left boxs">
+                  <div className="img_boxreview">
+                    <a href="#">
+                      <img src={this.state.url} alt="review_img1" />
```

Comment:
make sure this is absolute url, as our proxy site will be different 'http://localhost:3003'

Reply:
these are the profile pic urls I am getting from Matt, ie `https://avatars1.githubusercontent.com/u/18507767`

Reply:
mmm im going to say this isn't working for me but its trivial to have working for this PR ;p. also i would add as just always catch your promises (where you do the fetch), this isn't working because of a cors issue on matt's service, but it doesn't always thro. But add it to your list, to throw a catch in there.

Reply:
He is getting his images from lorempixel.com as well which has speculative loading times.  But I am also now running into a cors issues with his repo where it was just working yesterday.